formula/src/test/java/com/instacart/formula/FormulaRuntimeTest.kt

```diff
@@ -1352,6 +1352,123 @@ class FormulaRuntimeTest(val runtime: TestableRuntime, val name: String) {
             }
     }
 
+    // TODO: I'm not sure if this is the right behavior
+    @Test
+    fun `action termination events are ignored`() {
+        val formula = object : Formula<Boolean, Int, Int>() {
+            override fun initialState(input: Boolean): Int = 0
+
+            override fun Snapshot<Boolean, Int>.evaluate(): Evaluation<Int> {
+                return Evaluation(
+                    output = state,
+                    actions = context.actions {
+                        if (input) {
+                            val action = object : Action<Unit> {
+                                override fun start(send: (Unit) -> Unit): Cancelable {
+                                    return Cancelable {
+                                        send(Unit)
+                                    }
+                                }
+
+                                override fun key(): Any? = null
+                            }
+
+                            action.onEvent {
+                                transition(state + 1)
+                            }
+                        }
+                    }
+                )
+            }
+        }
+
+        val observer = runtime.test(formula)
+        observer.input(true)
+        observer.input(false)
+        observer.input(true)
+        observer.input(false)
+        observer.output { assertThat(this).isEqualTo(0) }
```

Comment:
Should the output value be equal to 2?

true - state (0) + 1
false - state = 1
true - state (1)  + 1
false - state = 2

Reply:
Good question, this is why the test name + comment are describing the situation
```kotlin
// TODO: I'm not sure if this is the right behavior
@Test
fun `action termination events are ignored`()
```